### PR TITLE
Documentation for Optimal Stopping Time

### PR DESCRIPTION
**Pull request by ashutosh-b-b**
- [ ] Writing a tutorial
- [ ]Writing into the index

### CODE REVIEW

**Comment by ashutosh-b-b** [comment]
@ChrisRackauckas can u have a look at the tutorial?

File: docs/src/examples/optimal_stopping.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+sigma(du,u,p,t)  = (du .= Diagonal(beta*u))
+
+tspan = (0.0 , T)
+N = 50
+dt = tspan[2]/(N - 1)
+```
+The final part is the payoff function :
+<p align="center">
+  <img src="https://raw.githubusercontent.com/ashutosh-b-b/github-doc-images/master/payoff_function.png">
+</p>
```
Comment (ChrisRackauckas) [inline docs/src/examples/optimal_stopping.md:24]:
```suggestion
![](https://raw.githubusercontent.com/ashutosh-b-b/github-doc-images/master/payoff_function.png)
```

File: docs/src/examples/optimal_stopping.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+T = 1.0
+u0 = fill(80.00 , d , 1) #Initial Stock Price
+#Defining the drift (f) and diffusion(sigma)
+f(du,u,p,t) = (du .= r*u)
+sigma(du,u,p,t)  = (du .= Diagonal(beta*u))
+
+tspan = (0.0 , T)
+N = 50
+dt = tspan[2]/(N - 1)
+```
```
Comment (ChrisRackauckas) [inline docs/src/examples/optimal_stopping.md:20]: tex showing what PDE this corresponds to?

Comment (ashutosh-b-b) [reply docs/src/examples/optimal_stopping.md:20]: PDE? you mean the black scholes model?

Comment (ChrisRackauckas) [reply docs/src/examples/optimal_stopping.md:20]: yes

Comment (ChrisRackauckas) [reply docs/src/examples/optimal_stopping.md:20]: See the Documenter docs for how TeX works. You use a ```math block, or ``. So inline TeX isn't hard at all with documentation, so we should make liberal use of it to explain the relationship between the code to the math.

Comment (ashutosh-b-b) [reply docs/src/examples/optimal_stopping.md:20]: Oh.  I ll add the explanation.

**Comment by ashutosh-b-b** [comment]
@ChrisRackauckas

**Pull request merged**